api_tests/src/images.spec.ts

```diff
@@ -0,0 +1,54 @@
+jest.setTimeout(120000);
+
+import { UploadImage, DeleteImage } from "lemmy-js-client";
+import {
+  alpha,
+  setupLogins,
+  unfollowRemotes,
+} from "./shared";
+import fs = require('fs');
+
+beforeAll(async () => {
+  await setupLogins();
+});
+
+afterAll(async () => {
+  await unfollowRemotes(alpha);
+});
+
+test("Upload image and delete it", async () => {
```

Comment:
All of the `let`s and `var`s in this function can be replaced with `const` since they aren't reassigned.

Reply:
Fixed, I thought let is the same as in Rust but apparently not.